docs/docs/tutorial.md

```diff
@@ -495,7 +495,6 @@ var CommentForm = React.createClass({
     // TODO: send request to the server
     this.refs.author.getDOMNode().value = '';
     this.refs.text.getDOMNode().value = '';
-    return;
```

Comment:
This change is actually going to throw off our line highlighting (see the things we have in the opening of the code block, should end up being `{3-13, 16-19}`)